src/Chapter1.hs

```diff
@@ -631,8 +643,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
+sumLast2 :: Int -> Int
+sumLast2 n = a + b
+  where 
+    a = mod n 10
+    b =div (mod (n-a) 100)  10
```

Comment:
I don't think it is necessary to subtract `a` in this one 🤔 
Also, in this function you need also to consider negative input (hint: check out `abs` function 😉 )